static/admin/config.yml

```diff
@@ -215,6 +215,293 @@ collections:
+                                    },
+                                    {
+                                        label: 'Image',
+                                        name: 'image',
+                                        widget: 'image',
+                                    },
+                                    {
+                                        label: 'Caption',
+                                        name: 'caption',
+                                        widget: 'string',
+                                        required: false,
+                                    },
+                                ],
+                        },
+                    ],
+            }
+          - {
+                label: 'Protocols',
+                name: 'protocols',
+                widget: 'object',
+                required: false,
+                collapsed: true,
+                fields:
+                    [
+                        {
+                            label: 'Diagrams',
+                            name: 'diagrams',
+                            widget: 'list',
+                            required: false,
+                            collapsed: true,
+                            fields:
+                                [
+                                    {
+                                        label: 'Title',
+                                        name: 'title',
+                                        widget: 'string',
+                                        required: false,
+                                    },
+                                    {
+                                        label: 'Image',
+                                        name: 'image',
+                                        widget: 'image',
+                                    },
+                                    {
+                                        label: 'Caption',
+                                        name: 'caption',
+                                        widget: 'string',
+                                        required: false,
+                                    },
+                                ],
+                        },
+                    ],
+            }
+          - {
+                label: 'Supplementary Information',
+                name: 'supplementary_info',
+                widget: 'list',
+                required: false,
+                fields:
+                    [
+                        {
+                            label: 'Acknowledgements',
+                            name: 'acknowledgements',
+                            widget: 'string',
+                            required: false,
+                        },
+                        {
+                            label: 'Supplementary Data',
+                            name: 'supplementary_data',
+                            widget: 'string',
+                            required: false,
+                        },
```

Comment:
nit: In call catalog 1.0, these are files, but we might want to wait for the new design to conform for 2.0.